lib/esc.js

```diff
@@ -72,6 +73,7 @@ var Controllers = {
       }
     },
     write: {
+      writable: true,
```

Comment:
Just curious, why is ```writable: true``` here? It doesn't hurt anything but ```write``` isn't on Devices so it doesn't ever get changed.